### DISCUSSION TITLE
Branch protection rules on a main branch

### DISCUSSION THREAD

**Discussion by Zejnilovic** [Homebrew/discussions#3643] [Tap maintenance and Homebrew development] [answered]
Hello,

I am trying to create a tap for our organization. In creation, I have used the `brew tap-new ORG/NAME`. This provides me with out of the box solution for bottles and runs tests. 

**My question is** - what protection rules can I set for the main branch if I am using "your" actions? So far, it seems that it directly pushes to the main branch and does not use the PR.

**Comment by SMillerDev** [comment, marked as answer]
You can set it only allow github tokens to push to your main branch.

**Reply by Zejnilovic** [reply]
I am not aware of how to do that in branch protection rules. 

Which of the bellow option would be that? 
<img width="782" alt="image" src="https://user-images.githubusercontent.com/17467768/189643487-f1df91e4-f5fb-47db-af78-f79c62602270.png">

But I guess I could create a team and add only code owners to it, and only them then could do such a task.
